packages/commands-http/test/request.ts

```diff
@@ -0,0 +1,77 @@
+import { BasicTest } from '@seagull/testing'
+import { expect } from 'chai'
+import 'chai/register-should'
+import * as fs from 'fs'
+import { skip, slow, suite, test, timeout } from 'mocha-typescript'
+import { Http, RequestConfig } from '../src'
+
+@suite('Http::Request')
+export class Test extends BasicTest {
```

Comment:
There is no spec for a different mode. Can we add one?

Reply:
Right now, commands are supposed to run within pure mode (yes, all other code will be untested). This should be discussed within the commands/architecture discussion.

Reply:
We could add something like 
```
originMode: IMode = Mode
  async before() {
    this.originMode = Mode
  }

  async after() {
    new SetMode('environment', this.originMode.environment).execute()
  }
``` in the test class and add `new SetMode('environment', 'cloud').execute()` within the spec to test the cloud functionality. What do you think?

Reply:
Another option is to call `executeCloud` explicitly without changing the mode.